pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1448,7 +1448,8 @@ func getCPUTopology(vmi *v1.VirtualMachineInstance) *CPUTopology {
 			sockets = vmiCPU.Sockets
 		}
 	}
-
+	// A default guest CPU topology is being set in API mutator webhook, if nothing provided by a user.
+	// However this setting is still required to handle situations when the webhook fails to set a default topology.
```

Comment:
I would misinterpret that. What we want to say is: "Not every version of kubevirt did that. We have to keep this to cope with these version during updates or with VMs which were started on such releases and are still running".